lib/menu/app.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
 						{ label: 'Toggle related files', command: 'appc:open-or-close-related' },
 						{ label: 'Add component...', command: 'appc:generate' },
 						{ label: 'Clean project', command: 'appc:clean' },
+						{ label: 'Screenshot of a device/simulator/emulator', command: 'appc:takeScreenshot' },
```

Comment:
Simply "Take Screenshot"? More slicky for a menu command. Devs should know where they are trying to do the screenshot from, based on the run-config selected in the UI.